sync/class.jetpack-sync-module-woocommerce.php

```diff
@@ -194,43 +194,118 @@ public function add_woocommerce_post_meta_whitelist( $list ) {
 
 	private static $wc_post_meta_whitelist = array(
 		//woocommerce products
-		'_stock_status',
+		// https://github.com/woocommerce/woocommerce/blob/8ed6e7436ff87c2153ed30edd83c1ab8abbdd3e9/includes/data-stores/class-wc-product-data-store-cpt.php#L21
```

Comment:
Instead of manually keeping this in sync with `Jetpack_Sync_Module_WooCommerce::$internal_meta_keys`, I think it would be better to use the `jetpack_sync_post_meta_whitelist` hook.

Reply:
I agree that this would be a good enhancement to add to WooCommerce, but I don't think it should block merging this change.